src/components/PressableWithSecondaryInteraction/index.tsx

```diff
@@ -89,42 +86,28 @@ function PressableWithSecondaryInteraction({
         return () => {
             element.removeEventListener('contextmenu', executeSecondaryInteractionOnContextMenu);
         };
-    }, [forwardedRef, onSecondaryInteraction, preventDefaultContextMenu, withoutFocusOnSecondaryInteraction]);
+    }, [ref, onSecondaryInteraction, preventDefaultContextMenu, withoutFocusOnSecondaryInteraction]);
 
-    const defaultPressableProps = _.omit(rest, ['onLongPress']);
     const inlineStyle = inline ? styles.dInline : {};
 
     // On Web, Text does not support LongPress events thus manage inline mode with styling instead of using Text.
     return (
         <PressableWithFeedback
+            // eslint-disable-next-line react/jsx-props-no-spreading
+            {...rest}
             wrapperStyle={StyleUtils.combineStyles(DeviceCapabilities.canUseTouchScreen() ? [styles.userSelectNone, styles.noSelect] : [], inlineStyle)}
-            onPressIn={onPressIn}
             onLongPress={onSecondaryInteraction ? executeSecondaryInteraction : undefined}
             pressDimmingValue={activeOpacity}
-            onPressOut={onPressOut}
             onPress={onPress}
```

Comment:
NAB: You removed `onPressIn` and `onPressOut` but not `onPress`. I know that `onPress` is a required value, so just pointing it out, maybe it is worth removing 😄

Reply:
Done!